src/phoenix/experimental/evals/evaluators.py

```diff
@@ -46,27 +50,48 @@ def evaluate(
             provide_explanation (bool, optional): Whether to provide an
             explanation.
 
+            use_function_calling_if_available (bool, optional): If True, use
+            function calling (if available) as a means to constrain the LLM
+            outputs. With function calling, the LLM is instructed to provide its
+            response as a structured JSON object, which is easier to parse.
+
             verbose (bool, optional): Whether to print verbose output.
 
         Returns:
             Tuple[str, Optional[str]]: The label and explanation (if provided).
         """
+        use_openai_function_call = (
+            use_function_calling_if_available
+            and isinstance(self._model, OpenAIModel)
+            and self._model.supports_function_calling
```

Comment:
this could be a separate function that's re-used I think. Centralizing it now might have a benefit because Claude2.1 and even some OSS models are getting this.

Reply:
Moving this into our `utils` sub-module will cause some circular import issues. Going to punt for now, but I agree we need to look at consolidation.